src/mocks/handler.ts

```diff
@@ -605,6 +609,38 @@ const getNewAccessTokenHandler = [
   }),
 ];
 
+const putChangeAgeHandler = [
+  http.put(SETTING_END_POINT.CHANGE_AGE, async ({ request }) => {
+    await new Promise((resolve) => setTimeout(resolve, 1000));
+    const { age } = (await request.json()) as {
+      age: keyof typeof AGE_RANGE_MAP;
+    };
```

Comment:
as로 하는 것 대신 put의 제너럴 타입으로 request.json 타입을 정하는 건 어떨까요?
```suggestion
  http.put<PathParams, { age: keyof typeof AGE_RANGE_MAP }>(SETTING_END_POINT.CHANGE_AGE, async ({ request }) => {
    await new Promise((resolve) => setTimeout(resolve, 1000));
    const { age } = await request.json();
```

Reply:
타입 선언하기 귀찮아서 as 때렸는데 `RequestData` 로 변경하겠습니다